homeassistant/components/homekit_controller/config_flow.py

```diff
@@ -218,15 +190,6 @@ async def async_step_zeroconf(self, discovery_info):
         self.context["title_placeholders"] = {"name": name}
 
         if paired:
-            old_pairings = await self.hass.async_add_executor_job(
-                load_old_pairings, self.hass
-            )
-
-            if hkid in old_pairings:
-                return await self.async_import_legacy_pairing(
```

Comment:
After this is removed does anything else call async_import_legacy_pairing? If not that function can be removed as well.

Reply:
Oops, removed in latest commit.